google/cloud/aiplatform/docker_utils/utils.py

```diff
@@ -28,6 +28,9 @@
 
 Package = collections.namedtuple("Package", ["script", "package_path", "python_module"])
 Image = collections.namedtuple("Image", ["name", "default_home", "default_workdir"])
+DEFAULT_HOME = "/home"
+DEFAULT_WORKDIR = "/usr/app"
+DEFAULT_MOUNTED_MODEL_DIRECTORY = "/tmp_cpr_local_model"
```

Comment:
How about just "/model"

Reply:
I prefer using a non-general name in case users' images already have files there since we don't provide a way to let users customize it.